src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -79,82 +79,94 @@ const PIAReview = () => {
-                  label="Select a role from the list"
-                  options={Object.keys(ApprovalRoles).map((role: string) => ({
-                    value: role,
-                    label: ApprovalRoles[role],
-                  }))}
-                  value={rolesSelect}
-                  changeHandler={(e) => setRolesSelect(e.target.value)}
-                />
-                <button
-                  className="bcgovbtn bcgovbtn__secondary mt-3"
-                  onClick={() => {
-                    reviewForm.programArea?.selectedRoles.push(
-                      ApprovalRoles[rolesSelect],
-                    );
-                    setRolesSelect('');
-                    stateChangeHandler(
-                      reviewForm.programArea?.selectedRoles,
-                      'programArea.selectedRoles',
-                    );
-                    piaStateChangeHandler(
-                      {
-                        programArea: {
-                          selectedRoles: reviewForm.programArea.selectedRoles,
-                        },
-                      },
-                      'review',
-                      true,
-                    );
-                  }}
-                >
-                  Add
-                </button>
-              </div>
-              <div className="p-2 col-md-2 d-flex justify-content-center align-items-center">
-                Or
-              </div>
-              <div className="p-2 col-md-5">
-                <InputText
-                  id="programArea"
-                  label={
-                    messages.PiaReviewHeader.ProgramAreaSection.Input
-                      .EnterRoleTitle.en
-                  }
-                  value={rolesInput}
-                  onChange={(e) => setRolesInput(e.target.value)}
-                />
-                <button
-                  className="bcgovbtn bcgovbtn__secondary mt-3"
-                  onClick={() => {
-                    reviewForm.programArea?.selectedRoles.push(rolesInput);
-                    stateChangeHandler(
-                      reviewForm.programArea.selectedRoles,
-                      'programArea.selectedRoles',
-                    );
-                    setRolesInput('');
-                    piaStateChangeHandler(
-                      {
-                        programArea: {
-                          selectedRoles: reviewForm.programArea.selectedRoles,
-                        },
-                      },
-                      'review',
-                      true,
-                    );
-                  }}
-                >
-                  Add
-                </button>
-              </div>
-            </div>
-            <div className="horizontal-divider mt-5 mb-5"></div>
+          <div
+            className={`${
+              pia.status !== PiaStatuses.FINAL_REVIEW && 'data-row'
+            }`}
+          >
+            {pia.status !== PiaStatuses.FINAL_REVIEW && (
```

Comment:
How about being explicit and mentioning only if one of MPO_Review or CPO_Review? NOT might be problematic in the COMPLETE status after FINAL_REVIEW.

Reply:
This shouldn't cause any problems because, from what I can see, this area is not even seen for a completed status.